### PR TITLE
Update QGIS master version to 3.8.0

### PR DESCRIPTION
**Pull request by SrNetoChan**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Updates master (stable branch to 3.8.0

### CODE REVIEW

**Comment by SrNetoChan** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+{% set version = "3.8.0" %}
 
 package:
   name: qgis
   version: {{ version }}
 
 source:
   url: https://qgis.org/downloads/qgis-{{ version }}.tar.bz2
-  sha256: b77790b47d978035e24d4652c5ad4227ae68205c056db54fa0fd2da6bf6f7b7d
+  md5: 81af2b33732ac4fa21ea2adf8bf72d4e
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Let's use the sha256 instead.

Comment (SrNetoChan) [reply recipe/meta.yaml]: The QGIS project provides the checksum in md5, not using it mean one need to download the package, check it with md5, and then create the sha256. that's why I changes it.

Comment (ocefpaf) [reply recipe/meta.yaml]: The bot does all that for us. We just need to implement a way for the bot to update the branches.

**Pull request merged**